jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -131,7 +132,8 @@ public Job build() {
                 _name = "cookjob";
             }
```

Comment:
Do we need to set the default value for `_isMeaCulpaRetriesDisabled` to false?

Alternatively, why are we using `Boolean` instead of `boolean`?